app/assets/GithubLogo.tsx

```
@@ -1,25 +1,25 @@
+
+const GithubLogo: React.FC<GithubLogoProps> = ({
+  svgClassName,
+  pathClassName,
+}) => (
+  // SVG source: https://github.com/logos
+  <svg
+    className={svgClassName}
+    // width="98" height="96"
+    viewBox="0 0 100 100"
+    xmlns="http://www.w3.org/2000/svg"
+  >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)